downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -2,79 +2,139 @@
+endif::container-install[]
+| Optional
+|
+
+| `aap_ca_cert_files_remote` 
+| `ca_tls_remote` 
+| Denote whether the CA certificate files are local to the installation program (`false`) or on the remote component server (`true`).
+| Optional
+| `false`
+
+| `aap_ca_cert_size`
+| 
+| Bit size of the internally managed CA certificate private key.
+| Optional
+| `4096`
+
+| `aap_ca_key_file` 
+| `ca_tls_key` 
+| Path to the key file for the CA certificate provided in `aap_ca_cert_file` (RPM) and `ca_tls_cert` (Container).
+// This content is used in RPM installation
+ifdef::aap-install[]
+For more information, see link:{URLInstallationGuide}/platform-system-requirements#optional_using_custom_tls_certificates[Optional: Using custom TLS certificates].
+endif::aap-install[] 
+// This content is used in Containerized installation
+ifdef::container-install[]
+For more information, see link:{URLContainerizedInstall}/ansible_automation_platform_containerized_installation#using-custom-tls-certificates_aap-containerized-installation[Using custom TLS certificates].
+endif::container-install[]
+| Optional
+| 
+
+| `aap_ca_passphrase_cipher`
+|
+| Cipher used for signing the internally managed CA certificate private key.
+| Optional
+| `aes256`
+
+| `aap_ca_regenerate`
+|
+| Denotes whether or not to re-initiate the internally managed CA certificate key pair.
+| Optional
+| `false`
+
+| `aap_service_cert_size` 
+|  
+| Bit size of the component key pair managed by the internal CA. 
+| Optional 
+| `4096`
+
+| `aap_service_regen_cert` 
+|  
+| Denotes whether or not to re-initiate the component key pair managed by the internal CA. 
+| Optional 
+| `false`
+
+| `aap_service_san_records` 
+|  
+| A list of additional SAN records for signing a service. These should be assigned to components in the inventory file as host variables rather than group or all variables. All strings should also contain their corresponding SAN option prefix such as `DNS:` or `IP:`. 
+| Optional 
+| `[]`
+
+| `backup_dest` 
+|  
+| Directory local to `setup.sh` for the final backup file. 
+| Optional 
+| The value defined in `setup_dir`.
+
+| `backup_dir` 
+| `backup_dir` 
+| Directory used to store backup files. 
+| Optional 
+| RPM = `/var/backups/automation-platform/`. Container = `~/backups`
+
+| `backup_file_prefix` 
+|  
+| Prefix used for the file backup name for the final backup file. 
+| Optional 
+| `automation-platform-backup`
+
+| `bundle_install` 
+| `bundle_install` 
+| Controls whether or not to perform an offline or bundled installation. Set this variable to `true` to enable an offline or bundled installation.
+| Optional 
+| `false`
+
+| `bundle_install_folder` 
+| `bundle_dir` 
+| Path to the bundle directory used when performing a bundle install. 
+| Required if `bundle_install=true` 
+| `false`
```

Comment:
The default is `/var/lib/ansible-automation-platform-bundle` for RPM installer, `<current dir>/bundle` for containerized.